src/scripts/ugg.js

```diff
@@ -0,0 +1,88 @@
+/*
+*	U.GG
+*/
+chrome.runtime.sendMessage({
+    from:    'content',
+    subject: 'showPageAction'
+});
+
+chrome.runtime.onMessage.addListener(function(msg, sender, response) {
+    if ((msg.from === 'popup') && (msg.subject === 'guide')) {
+
+		var BLOCKS = {
+			title: null, 
+			items: []
+		};
+		var title = stripURL($('.champion-image').attr('src')) + " - U.GG";
+		BLOCKS.title = title;
+
+		for (i = 0; i < 2; i++) {
```

Comment:
Lets combine the two iterative functions as they both perform the same job, we can iterate on the set of fields.